tests/test_pyproject_pep660.py

```diff
@@ -57,8 +58,31 @@ def test_pep660_pip_isolated(isolated, isolate):
     value = isolated.execute("import simplest; print(simplest.square(2))")
     assert value == "4.0"
 
-    location = isolated.execute("import simplest; print(*simplest.__path__)")
-    assert location == str(Path.cwd() / "src/simplest")
+    location_str = isolated.execute(
+        "import simplest; print(*simplest.__path__, sep=';')"
+    )
+    locations = [Path(s).resolve() for s in location_str.split(";")]
+
+    # First path is from the python source
+    python_source = Path("src/simplest").resolve()
+    assert any(x.samefile(python_source) for x in locations)
+
+    # Second path is from the CMake install
+    cmake_install = isolated.platlib.joinpath("simplest").resolve()
+    assert any(x.samefile(cmake_install) for x in locations)
 
     location = isolated.execute("import simplest; print(simplest.__file__)")
-    assert location == str(Path.cwd() / "src/simplest/__init__.py")
+    # The package file is defined in the python source and __file__ must point to it
+    assert Path("src/simplest/__init__.py").resolve().samefile(Path(location).resolve())
+
+    location = isolated.execute(
+        "import simplest._module; print(simplest._module.__file__)"
+    )
+    if sys.version_info < (3, 8, 7):
+        import distutils.sysconfig  # pylint: disable=deprecated-module
+
+        ext_suffix = distutils.sysconfig.get_config_var("EXT_SUFFIX")
+    else:
+        ext_suffix = sysconfig.get_config_var("EXT_SUFFIX")
+    module_file = cmake_install / f"_module{ext_suffix}"
+    assert module_file.samefile(Path(location).resolve())
```

Comment:
Uhm, this seems not right:
```
location   = 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-runneradmin\\pytest-0\\test_pep660_pip_isolated_False0\\venv\\lib\\site-packages\\simplest\\_module.pyd'
location_str = 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-runneradmin\\pytest-0\\test_pep660_pip_isolated_False0\\pkg\\...ata\\Local\\Temp\\pytest-of-runneradmin\\pytest-0\\test_pep660_pip_isolated_False0\\venv\\lib\\site-packages\\simplest'
locations  = [WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-runneradmin/pytest-0/test_pep660_pip_isolated_False0/p...in/AppData/Local/Temp/pytest-of-runneradmin/pytest-0/test_pep660_pip_isolated_False0/venv/Lib/site-packages/simplest')]
module_file = WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-runneradmin/pytest-0/test_pep660_pip_isolated_False0/venv/Lib/site-packages/simplest/_module.cp37-win_amd64.pyd')
```

Reply:
My guess is it might be a bug in CMake's FindPython not getting the correct `EXT_SUFFIX`; it's important to use `distutils.sysconfig` instead of `sysconfig` before Python 3.8.7 fixed a bug in `sysconfig`.

I'm not too worried about it, we can just check for both (at least before 3.8.7). If you don't include the SOABI, it still imports. Maybe we should report it to CMake.

Reply:
I haven't run the tests in a container to verify, but at least the failing tests report it as such. `3.7` on Windows, not on the linux/macos environments.

Reply:
```suggestion
    if sys.version_info < (3, 8, 7):
        import distutils.sysconfig  # pylint: disable=deprecated-module

        ext_suffix = distutils.sysconfig.get_config_var("EXT_SUFFIX")
    else:
        ext_suffix = sysconfig.get_config_var("EXT_SUFFIX")
    module_file = cmake_install / f"_module{ext_suffix}"
    
    # Windows FindPython may produce the wrong extension
    if sys.version_info < (3, 8, 7) and sys.platform.startswith("win") and not module_file.is_file():
        module_file = cmake_install / "_module.pyd"
        
    assert module_file.samefile(Path(location).resolve())
```